testutils/iotlab.py

```diff
@@ -62,7 +63,7 @@ def board_from_iotlab_node(iotlab_node):
                 "name of format "
                 "<node-name>.<site-name>.iot-lab.info"
             )
-        iotlab_node_name = match.group(1)
+        iotlab_node_name = iotlab_node_name = match.group(1)
```

Comment:
Huh? You assign the node name then assign it again? I think this is some relic of an older change ;-).

Reply:
copy and paste issue... Still need some time to get it clean.

Reply:
```suggestion
        iotlab_node_name = match.group(1)
```

Reply:
@MrKevinWeiss :eyes: